config/cp4waiops/install-aimgr/values.yaml

```diff
@@ -4,9 +4,6 @@
 ###################################
 
 spec:
-  ## AI Manager catalog source image
-  ##
-  #imageCatalog: icr.io/cpopen/ibm-operator-catalog:latest
```

Comment:
I think the `spec.imageCatalg` is needed here, because this ensures that the install-aimgr chart can be deployed independently, w/o all-in-one chart.

Reply:
In current release-3.4, the `imageCatalog` and matching `catalogSource` are moved to cp-shared , and they are not involved/referenced in aimanager any more.  Will discuss this later.